backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -80,6 +80,7 @@ protected void releaseLiquibaseLock(DatabaseConfig config)
       if (!existsStatus) {
         LOGGER.info("Table database_change_log_lock does not exists in DB");
         LOGGER.info("Proceeding with liquibase assuming it has never been run");
+        return;
```

Comment:
This seems unrelated, no?

Reply:
Yes but since it is just one line and related to jdbi changes so i thought it will works here. :)